openlibrary/plugins/upstream/data.py

```diff
@@ -5,22 +5,17 @@
 from infogami.utils import delegate
 from infogami.utils.view import public
 
-import simplejson
-
-from six.moves import urllib
+import requests
 
 
 IA_BASE_URL = config.get('ia_base_url')
 
 
-def wget(url):
-    return urllib.request.urlopen(url).read()
-
 
 def get_ol_dumps():
     """Get list of all archive.org items in the ol_exports collection uploaded by archive.org staff."""
     url = IA_BASE_URL + '/advancedsearch.php?q=(ol_dump+OR+ol_cdump)+AND+collection:ol_exports&fl[]=identifier&output=json&rows=1000'
-    d = simplejson.loads(wget(url))
+    d = requests.get(url).json()
     return sorted(doc['identifier'] for doc in d['response']['docs'])
```

Comment:
```suggestion
    docs = requests.get(url).json()['response']['docs']
    return sorted(doc['identifier'] for doc in docs)
```
I am almost never a fan of single letter variable names.  They feel so old school to me.